OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseBaseActivity.java

```diff
@@ -65,6 +74,8 @@ protected void onCreate(Bundle arg0) {
             bar.setDisplayHomeAsUpEnabled(true);
             bar.setIcon(android.R.color.transparent);
         }
+        errorNotification = new OverlayErrorNotification(progressWheel);
```

Comment:
In all the places where we have used name `errorNotification` for a field which is initializing with `OverlayErrorNotification`, `overlayErrorNotification`  name will be more clean and easy to understand the purpose.